### PR TITLE
search bars task

### PR DESCRIPTION
**Pull request by mttony**
[DEMO LINK](https://mttony.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://mttony.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
```
Comment (liliia-livandovskaya) [inline src/style.css:15]: font should be set in `html` selector tad and inherited where needed

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon--small {
+  position: absolute;
```
Comment (liliia-livandovskaya) [inline src/style.css:90]:
```suggestion
```
this is duplicated one

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,46 @@
...
+        <input 
+          class="large_input"
+          name="large_input"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
+    </form>
+    <form
```
Comment (liliia-livandovskaya) [inline src/index.html:35]:
```suggestion
    </form>
    
    <form 
```

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Great job! Let's work on a fix

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
image

**Review by vlad-demchuk — APPROVED** [review]
Just check order of css properties🙂
https://codeguide.academy/html-css.html#css-order

**Pull request closed without merging**